pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -311,6 +311,16 @@ func (r *ReconcileOperConfig) Reconcile(ctx context.Context, request reconcile.R
 	// Fill all defaults explicitly
 	network.FillDefaults(&newOperConfig.Spec, prev, mtu)
 
+	//TO-REMOVE - 5 lines
```

Comment:
Look for //To-Remove comments, these are lines I planned to remove, decided what you want to keep and in any case remove the comments.

Reply:
done